plugins/filebrowser/tpl/editorbrowser.html

```diff
@@ -5,7 +5,7 @@
 <html lang="en">
     <head>
         <title>Filebrowser</title>
-        <link rel="stylesheet" href="<?php echo "{$pth['folder']['corestyle']}stylesheet.css"?>" type="text/css">
+        <link rel="stylesheet" href="<?php echo "{$pth['folder']['corestyle']}xhstyles.css"?>" type="text/css">
         <style type="text/css">body {padding: 0; margin: 0; overflow: auto;}</style>
```

Comment:
I'm not sure if that should be changed. It's probably best to keep this out of this PR, because it is not related.